internal/models/user.go

```diff
@@ -27,6 +29,7 @@ type CreateUserRequestModel struct {
 	LastName    string `json:"last_name" validate:"required"`
 	UserName    string `json:"username" validate:"required"`
 	PhoneNumber string `json:"phone_number"`
+	Role        string `json:"role"`
```

Comment:
is this okay?
a user can specify their role while registering?

Reply:
What do you suggest?